egs/aurora4/asr1/RESULTS.md

```diff
@@ -1,4 +1,95 @@
+# Transformer result(default transformer with n_average = 10, epoch = 100)
+  
+  - Environments (obtained by `$ get_sys_info.sh`)
+    - date: `Thu Jun 13 01:29:55 EDT 2019`
+    - system information: `Linux a09 4.9.0-4-amd64 #1 SMP Debian 4.9.65-3+deb9u1 (2017-12-23) x86_64 GNU/Linux`
+    - python version: `Python 3.6.8 :: Anaconda, Inc.
+    - espnet version: `espnet 0.3.1`
+    - chainer version: `chainer 5.0.0`
+    - pytorch version: `pytorch 1.0.0`
+    - Git hash: `04b9e1a71b3272cdf6502e40679ab17f3f3f22f6`
+  - Model files (archived to model.tar.gz by `$ pack_model.sh`)
+    - model link: 
```

Comment:
Can you update a model to the Google drive I sent an invitation and put the model link here?